docs/source/techniques/ISISPowder-OSIRIS-v1.rst

```diff
@@ -0,0 +1,307 @@
+                          output_directory=output_dir,
+                          config_file=config_dir)
+
+Methods
+--------
+The following methods can be executed on a OSIRIS object:
+
+- :ref:`run_diffraction_focusing_osiris_isis-powder-diffraction-ref`
+
+For information on creating a OSIRIS object see:
+:ref:`creating_osiris_object_isis-powder-diffraction-ref`
+
+.. _run_diffraction_focusing_osiris_isis-powder-diffraction-ref:
+
+run_diffraction_focusing
+^^^^^^^^^^^^^^^^^^^^
+The *diffraction_focusing* method allows a user to process a series of runs into a
+focused dSpacing workspace. Whilst processing the runs the scripts can apply any corrections
+the user enables.
+
+On OSIRIS the following parameters are required when executing *create_vanadium*:
+
+- :ref:`calibration_mapping_file_osiris_isis-powder-diffraction-ref`
+- :ref:`vanadium_normalisation_osiris_isis-powder-diffraction-ref`
+- :ref:`subtract_empty_instrument_osiris_isis-powder-diffraction-ref`
+- :ref:`merge_drange_osiris_isis-powder-diffraction-ref`
+
+Example
+=======
+..  code-block:: python
+
+  # Notice how the filename ends with .yaml
+  cal_mapping_file = r"C:\path\to\cal_mapping.yaml"
+
+  osiris_example.run_diffraction_focusing(run_number="119977-119988",
+                                          merge_drange=True,
+                                          subtract_empty_instrument=True,
+                                          vanadium_normalisation=True,
+                                          calibration_mapping_file=cal_mapping_file)
+.. _calibration_mapping_osiris-isis-powder-ref:
+
+Calibration Mapping File
+-------------------------
+The calibration mapping file holds the mapping between
+run numbers, current label, offset filename and the empty
+and vanadium numbers.
+
+For more details on the calibration mapping file see:
+:ref:`cycle_mapping_files_isis-powder-diffraction-ref`
+
+The layout on OSIRIS should look as follows for each block
+substituting the below values for appropriate values:
+
+.. code-block:: yaml
+  :linenos:
+
+  1-120:
+    label: "1_1"
+    offset_file_name: "offset_file.cal"
+    vanadium_drange1 : "13"
+    vanadium_drange2 : "14"
+    vanadium_drange3 : "15"
+    vanadium_drange4 : "16"
+    vanadium_drange5 : "17"
+    vanadium_drange6 : "18"
+    vanadium_drange7 : "19"
+    vanadium_drange8 : "20"
+    vanadium_drange9 : "21"
+    vanadium_drange10 : "22"
+    vanadium_drange11 : "23"
+    vanadium_drange12 : "24"
+    vanadium_run_numbers : "13-24"
+    empty_drange1 : "1"
+    empty_drange2 : "2"
+    empty_drange3 : "3"
+    empty_drange4 : "4"
+    empty_drange5 : "5"
+    empty_drange6 : "6"
+    empty_drange7 : "7"
+    empty_drange8 : "8"
+    empty_drange9 : "9"
+    empty_drange10 : "10"
+    empty_drange11 : "11"
+    empty_drange12 : "12"
+    empty_run_numbers : "1-12"
+
+for each set of run numbers the associated vanadium and empty runs for each drange must be listed for their drange.
```

Comment:
I dont understand what you mean by "each drange must be listed for their drange"

Reply:
could we update the doc to your comment? It is much clearer and easier to understand